xarray/backends/pseudonetcdf_.py

```diff
@@ -85,3 +88,65 @@ def get_encoding(self):
 
     def close(self):
         self._manager.close()
+
+
+def open_backend_dataset_pseudonetcdf(
+    filename_or_obj,
+    mask_and_scale=False,
+    decode_times=None,
+    concat_characters=None,
+    decode_coords=None,
+    drop_variables=None,
+    use_cftime=None,
+    decode_timedelta=None,
+    mode=None,
+    lock=None,
+    **format_kwargs,
+):
+
+    store = PseudoNetCDFDataStore.open(
+        filename_or_obj, lock=lock, mode=mode, **format_kwargs
+    )
+
+    with close_on_error(store):
+        vars, attrs = store.load()
+        file_obj = store
+        encoding = store.get_encoding()
+
+        vars, attrs, coord_names = conventions.decode_cf_variables(
+            vars,
+            attrs,
+            mask_and_scale=mask_and_scale,
+            decode_times=decode_times,
+            concat_characters=concat_characters,
+            decode_coords=decode_coords,
+            drop_variables=drop_variables,
+            use_cftime=use_cftime,
+            decode_timedelta=decode_timedelta,
+        )
+
+        ds = Dataset(vars, attrs=attrs)
+        ds = ds.set_coords(coord_names.intersection(vars))
+        ds._file_obj = file_obj
+        ds.encoding = encoding
+
+    return ds
+
+
+open_dataset_parameters = (
```

Comment:
I'm surprised by the "**format_kwargs" entry, do we have a use for it?

@aurghs I guess the reason you define `open_dataset_parameters` explicitly is to allow kwargs, but I don't think we need to add it here.

Reply:
I don't know it format_kwargs is used. I would prefer to keep it.

Reply:
I was referring to the string inside `open_dataset_parameters`, line 147.

Reply:
It's not used, maybe it could be missleading, I remove it.